src/test-utils/override-fs-with-fakes.ts

```diff
@@ -50,6 +58,8 @@ export const getOverrideFsWithFakes = () => {
       rm: root.promises.rm,
       access: root.promises.access,
       copy: async (src, dest) => { throw new Error(`Tried to copy '${src}' to '${dest}'. Copying is not yet supported`); },
+      ensureDir: async (path, opts) => ensureDirSync(path, opts),
+      ensureDirSync,
```

Comment:
I'm not sure if these should be deprecated, or even left out of `fs.injectable`, as using them is mostly sign of trouble, and/or the responsibility of other abstraction. WDYT?